include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -173,31 +175,31 @@ class alignment_algorithm :
      * |space (begin positions) |\f$ O(n*m) \f$    |\f$ O(n*k) \f$     |
      * |space (alignment)       |\f$ O(n*m) \f$    |\f$ O(n*k) \f$     |
      */
-    template <indexed_sequence_pair_range indexed_sequence_pairs_t>
+    template <indexed_sequence_pair_range indexed_sequence_pairs_t,
+              typename callback_t>
```

Comment:
put on the line above :nail_care:

Reply:
can I do this in the upcoming PRs since I am anyway touching these lines?